src/ibei/models.py

```diff
@@ -356,17 +377,41 @@ def efficiency(self) -> astropy.units.Quantity[astropy.units.dimensionless_unsca
 @attrs.frozen
 class DeVosSolarcell(SQSolarcell):
     """
-    DeVos single-junction solar cell
+    DeVos model of single-junction solar cell
 
     This class implements a solar cell as described by
     DeVos :cite:`9780198513926` Ch. 6.
 
-    An DeVosSolarcell is instantiated with a :class:`dict` having keys
-    identical to the class's public data attributes. Each key's value
-    must satisfy the constraints noted with the corresponding public data
-    attribute. Dictionary values can be some kind of numeric type or of
-    type :class:`astropy.units.Quantity` so long as the units are
-    compatible with what's listed.
+    Parameters
+    ----------
+    bandgap:
+        Bandgap of solar cell material.
+    solar_temperature:
+        Temperature of the sun.
```

Comment:
I am repeating this documentation from the parent class. I would like to just inherit it instead.